source/com.microsoft.tfs.core/META-INF/MANIFEST.MF

```diff
@@ -20,8 +20,16 @@ Bundle-ClassPath: com.microsoft.tfs.core.jar,
  libs/jackson-core-2.4.1/jackson-core-2.4.1.jar,
  libs/jackson-core-2.4.1/jackson-databind-2.4.1.jar,
  libs/annotation-detector-3.0.4/annotation-detector-3.0.4.jar,
- libs/java-auth-core/java-auth-core-0.1.0.jar,
- libs/oauth2-useragent/oauth2-useragent-0.7.1.jar
+ libs/jna/jna-4.2.1.jar,
+ libs/jna/jna-platform-4.2.1.jar,
+ libs/oauth2-useragent/oauth2-useragent-0.8.1.jar,
+ libs/slf4j/slf4j-api-1.7.19.jar,
+ libs/slf4j/slf4j-log4j12-1.7.19.jar,
+ libs/azure-interactive-login/aad-interactive-auth-1.0.1-SNAPSHOT.jar,
```

Comment:
Auth library now fall back to MSOpenTech's SWT browser if JavaFx browser won't come up.